ape_ens/utils.py

```diff
@@ -0,0 +1,42 @@
+import idna
+from ape.exceptions import ConversionError
+from eth_utils import keccak
+
+
+class InvalidName(ConversionError):
+    pass
+
+
+def normalize_name(name: str) -> str:
```

Comment:
Add note about copying from web3py, as well as using EIP-162 to design functionality.